.github/workflows/ci.yml

```diff
@@ -72,6 +71,28 @@ jobs:
           cd ..
           echo "LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$PWD/usr/lib" >> $GITHUB_ENV
           echo "TOOLCHAIN_OPTION=$TOOLCHAIN_OPTION -DCMAKE_PREFIX_PATH=$PWD/usr" >> $GITHUB_ENV
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          driver: docker
+          id: builder
+      - name: Prepare OpenSSH Server Container for Testing
+        uses: docker/build-push-action@v2
+        with:
+          builder: builder
+          pull: true
+          load: true
+          push: ${{ github.event_name == 'push' }}
```

Comment:
Do I understand it correctly that this makes it only push the image for commits on master branch?

Reply:
Correct